src/Widgets/DataEmptyWidget/DataEmptyWidgetComponent.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  ContentTag,
+  DataScope,
+  isInPlaceEditingActive,
+  provideComponent,
+  // @ts-expect-error TODO: remove once officially released
+  useDataScope,
+} from 'scrivito'
+import { DataEmptyWidget } from './DataEmptyWidgetClass'
+import { EditorNoteOrNull } from '../../Components/EditorNoteOrNull'
+import { useState } from 'react'
+
+provideComponent(DataEmptyWidget, ({ widget }) => {
+  const dataScope: DataScope | undefined = useDataScope()
+  const [showAnyway, setShowAnyway] = useState(false)
+
+  if (!dataScope) {
+    return <EditorNoteOrNull>No data scope found!</EditorNoteOrNull>
+  }
+
+  if (dataScope.isEmpty()) {
+    return <ContentTag content={widget} attribute="content" />
+  }
+
+  if (!isInPlaceEditingActive()) return null
+
+  return (
+    <>
+      <EditorNoteOrNull>
+        The data scope is currently not empty.{' '}
+        {showAnyway ? (
+          <button
+            className="btn btn-sm btn-outline-secondary"
+            onClick={() => setShowAnyway(false)}
+          >
+            Hide content
+          </button>
+        ) : (
+          <button
+            className="btn btn-sm btn-outline-secondary"
+            onClick={() => setShowAnyway(true)}
+          >
+            Show content anyway
+          </button>
+        )}
+      </EditorNoteOrNull>
+      {showAnyway ? <ContentTag content={widget} attribute="content" /> : null}
```

Comment:
* [ ] FYI I'm not convinced the "show anyway" UX is what we want. Given that
  * The message + button take up more space than a typical "no items" message, so it often *increases* UI clutter instead of reducing it
  * It requires switching to preview for WYSIWYG anyway
  
  we could simply always show the content in edit mode. The conditional nature could be highlighted by one or more of
  * opacity
  * outline
  * background
  * widget label

Reply:
Nice idea with the `opacity`.